zips/draft-coinholder-directed-grants.md

```diff
@@ -0,0 +1,259 @@
+
+TBD: specify the rest of the overall voting process.
+
+## Participation and Voting Thresholds
+
+A proposal MUST have at least 420,000 ZEC participating to be eligible for approval, as specifed in ZIP 1016. [^zip-1016]
+
+A proposal is approved if a simple majority of participating votes are “Yes”.
+
+Each proposal vote is independent, and coins MAY be reused across proposals in the same cycle.
+
+## Proposal Funding and Disbursement
+
+After votes are complete, the Administrator MUST publish final voting results within seven days. The results MUST include total participation, per-proposal tallies, and data necessary for independent auditing.
+
+The Administrator MUST produce a ranked list of approved proposals based on the percentage of “Yes” votes.
+
+Funds MUST be allocated starting from the highest-ranked approved proposal until the treasury is exhausted.
+
+If insufficient funds remain to fully fund the next approved proposal, the Key-Holder Organizations MAY partially fund it up to the remaining balance.
+
+Approved proposals not funded or partially funded MAY be resubmitted in future quarters.
+
+Key-Holder Organizations MUST execute payments using the NU 6.1 multisignature scheme and MUST publish transaction identifiers.
+
+The Administrator MUST publish a disbursement report detailing allocated amounts, partial disbursements, and executed transactions.
+
+## Quarterly Timeline
+
+| Phase                        | Approximate Duration  | Description |
+|------------------------------|-----------------------|-------------|
+| Public Call for Proposals    | ~ 1 month             | The Administrator or its delegate announces an open call via community channels. Applicants prepare and submit completed-work proposals for consideration. |
+| Review and Discussion Period | ≥ 30 days             | Proposals are publicly discussed and evaluated. The Administrator or its delegate reviews submissions for eligibility and publishes a final list of qualified proposals. |
+| Registration Period          | ≥ 10 days             | Coinholders register their balances for voting by creating eligible Orchard notes within a defined block range (as described in Section 4.1). |
+| Coinholder Vote on Proposals | ~ 2 weeks             | Coinholders vote on eligible proposals using the process defined in Section 4. The Administrator or its delegate verifies and tallies the results. |
+| Vote Tallying                | After voting concludes | The Administrator or its delegate tallies all votes. For shielded voting, tallying is performed using the Coin Voting 2.0 Audit Tool [^zcash-vote-audit] to aggregate verified ballots submitted to the voting chain. For transparent voting, tallying is performed by verifying signed messages and corresponding transparent payments received at the published Orchard address using the IVK. Shielded and transparent results are combined into a unified tally prior to publication. |
```

Comment:
Most of the content here should probably not be in this table, but should instead be in the relevant specification sections instead. 

The first sentence can stay here, the second sentence belongs in the shielded voting method section, the third in the transparent voting method section, and the final sentence belongs in Voting Process.